llm_bench/python/convert.py

```diff
@@ -175,8 +177,8 @@ def convert_optimum_causallm_base(model, args):
     )
     if "decoder_with_past_model" in models_and_onnx_configs:
         models_and_onnx_configs = {"model": models_and_onnx_configs["decoder_with_past_model"]}
-    if args.bettertransformer:
```

Comment:
now it is not possible to use only bettertransformer without stateful?

Reply:
Correct.